### PR TITLE
101065 Spotfix for Template-Loading Error

### PR DESCRIPTION
**Pull request by ggwicz**
**Ticket:** [**#101065**](http://central.tri.be/issues/101065)

### CODE REVIEW

**Comment by ggwicz** [comment]
This one's ready for review 🔍

File: src/functions/template-tags/general.php [hunk 1]
```diff
@@ -1620,9 +1618,7 @@ function tribe_get_default_value( $field ) {
 	 */
 	function tribe_get_render_context( $query = null ) {
 
-		if ( ! $wp_query = tribe_get_global_query_object() ) {
-			return;
-		}
+		global $wp_query;
```
Comment (bordoni) [inline src/functions/template-tags/general.php]: Why global here?

File: src/functions/template-tags/general.php [hunk 2]
```diff
@@ -1242,9 +1242,7 @@ function tribe_events_template_data( $event = null, array $additional = null ) {
 	 **/
 	function tribe_include_view_list( $args = null, $initialize = true ) {
 
-		if ( ! $wp_query = tribe_get_global_query_object() ) {
-			return;
-		}
+		global $wp_query;
```
Comment (bordoni) [inline src/functions/template-tags/general.php]: Why global here?

**Comment by ggwicz** [comment]
Addressed Gustavo's questions in Slack [here](https://tribe.slack.com/archives/C0258NY1C/p1520441954000980)—proceeding with merge.

**Pull request merged**